packages/rule-tester/src/types/DependencyConstraint.ts

```diff
@@ -1,4 +1,11 @@
-import type { RangeOptions } from 'semver';
+// `Options` and `RangeOptions` are defined in the 'semver' package.
+// We redeclare them here to avoid a peer dependency on that package:
+export interface Options {
+  loose?: boolean | undefined;
+}
+export interface RangeOptions extends Options {
+  includePrerelease?: boolean | undefined;
```

Comment:
[Style] I'm not a huge fan of general-purpose names like `Options`, especially when they don't pertain to the main export of a package. Since it's just two properties I think it's reasonable to have a single interface exported. Is there a strong reason to keep `Options` standalone?

```suggestion
export interface RangeOptions extends Options {
  includePrerelease?: boolean | undefined;
  loose?: boolean | undefined;
```

Reply:
No reason -- these names are verbatim from `@types/semver`.

We can condense it.